src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c

```diff
@@ -397,6 +407,9 @@ int xocl_hot_reset(struct xocl_dev *xdev, u32 flag)
 		xocl_reset_notify(xdev->core.pdev, false);
 
 		xocl_drvinst_set_offline(xdev->core.drm, false);
+
+		//Reset Virtualization registers
+		(void) xocl_mb_read_p2p_addr(xdev);
```

Comment:
Why does it have to refresh bridge address table after resetting FPGA?

Reply:
Need to configure these PCIe extended registers again

Reply:
The question is why you need to configure it again? We do not reset the on-device pcie bridge, so why it lost the configuration?